Codegen/src/Generators/SignaturesGenerator.cs

```diff
@@ -57,8 +57,8 @@ public void Generate(CompilationUnit compilationUnit, ColumnsLayout columns = Co
 
             var sourceFile = compilationUnit.ProgramClassDocumentSnapshot.Root;
             sourceFile.AcceptASTVisitor(new ExportToDependency());
-            bool insideFormalizedComment = false;
             bool insideMultilineComment = false;
+            bool insideMultilineCommentNew = false;
```

Comment:
I think you can the code clearer using a boolean `shouldWriteLine` instead of `insideMultilineCommentNew` which is not meaningful.
The code would look like something like this :
```csharp
bool shouldWriteLine = true;
bool insideMultilineComments = false;
if (line is singleLineComment) shouldWriteLine = false;
if (line is multilineCommentStart)
{
    shouldWriteLine = false;
    insideMultilineComment = true;
}
if (line is multilineCommentStop)
{
    shouldWriteLine = false;
    insideMultilineComment = false;
}

...

if (shouldWriteLine && !insideMultilineComments)
{
    WriteLine
    shouldWriteLine = true;
}
```